scripts/apitypings/main.go

```diff
@@ -288,7 +288,7 @@ func (g *Generator) generateOne(m *Maps, obj types.Object) error {
 			// type <Name> string
 			// These are enums. Store to expand later.
 			m.Enums[obj.Name()] = obj
-		case *types.Map:
+		case *types.Map, *types.Array, *types.Slice:
```

Comment:
Is this still needed?

Reply:
Yes.

Reply:
Makes sense since `typescriptType` supports slices.